src/components/signup/SmsForm.js

```diff
@@ -192,7 +194,7 @@ const SMSAction = ({
     if (showWait) {
       setTimeout(() => {
         setWait(false)
-      }, 10000)
+      }, smsRateLimit)
```

Comment:
lets change the message we show to the user to include also a countdown similar to this
![image](https://user-images.githubusercontent.com/671095/121899089-c1e1fc00-cd2c-11eb-81bf-cb212f6f7017.png)